recipes/cfgfile/0.2.8.2.1/conanfile.py

```diff
@@ -0,0 +1,66 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+
+class CfgfileConan(ConanFile):
+    name = "cfgfile"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/igormironchik/cfgfile.git"
+    license = "MIT"
+    description = "Header-only library for reading/saving configuration files with schema defined in sources."
+    generators = "cmake"
+    topics = ("conan", "cfgfile", "configuration")
+    settings = "os", "arch", "compiler", "build_type"
```

Comment:
```suggestion
    settings = "os", "arch"
```
Installer doesn't require compiler and build_type

Reply:
Corresponding to Conan's documentation ([this one](https://github.com/conan-io/conan-center-index/blob/master/docs/faqs.md#why-installer-packages-remove-some-settings-from-their-package-id)) `build_type` is required, as this library provides executable tool. Not sure about `compiler`, but use `compiler` setting in `vaildate` method. And I guided to `CMake` recipe...